ke/compiler.py

```diff
@@ -217,6 +222,14 @@ def compile_macro(macro, macros):
     return macros[macro.name]
 
 
+def compile_multi_range(range, _):
```

Comment:
Maybe make that `>` and not `>=`?

Reply:
Also, this test should probably happen _after_ parsing to `int` (otherwise e.g. `"2" >= "12"`, woops :)